introduction.md

```diff
@@ -1 +1,5 @@
-Create a new file called introduction.md introducing yourself to the dojo. Make sure this file contains an introduction, then add, commit and push your change to the fork. my favorite Code School path is JavaScript.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?